### ISSUE TITLE
MiFlora HHCCJCY01 firmware version in BLE data?

### ISSUE THREAD

**Issue by ACrazyConcept** [custom-components/ble_monitor#526]
Do you know if the firmware version is in the BLE data for the MiFlora HHCCJCY01 Plant sensor? E.g. '3.3.1'

**Comment by Ernst79** [comment]
No, it does not send it passively. Perhaps when making a connection, but that is not supported in BLE monitor
